__tests__/TestRunner.ts

```diff
@@ -144,260 +144,274 @@ export type TestRunnerResult = {
     updateMarket?: Market;
 };
 
-export function testClient(spec: TestSpec) {
+/**
+ * Test a client given a spec.
+ *
+ * @param spec test specification
+ * @param [next] optional,  callback to execute when tests are complete
+ */
+export function testClient(spec: TestSpec, next?: (...args: any[]) => void) {
```

Comment:
The problem: In order to test two different variations of constructor arguments to `BitfinexClient` I wanted to be able to run `testClient()` once, wait for it to complete, then run `testClient()` again with other constructor arguments. So I just wrapped the entire test in one `describe()` and used `after()` to call the callback (if provided)

the only change I made here